tests/paged_roles_data.js

```diff
@@ -0,0 +1,324 @@
+    name: 'myRole-49',
+    id: 'myRoleId-49',
+    description: 'myDescription-49'
+  }, {
+    name: 'myRole-50',
+    id: 'myRoleId-50',
+    description: 'myDescription-50'
+  } ],
+  roles_page_2: [ {
+    name: 'myRole-51',
+    id: 'myRoleId-51',
+    description: 'myDescription-51'
+  }, {
+    name: 'myRole-52',
+    id: 'myRoleId-52',
+    description: 'myDescription-52'
+  }, {
+    name: 'myRole-53',
+    id: 'myRoleId-53',
+    description: 'myDescription-53'
+  }, {
+    name: 'myRole-54',
+    id: 'myRoleId-54',
+    description: 'myDescription-54'
+  }, {
+    name: 'myRole-55',
+    id: 'myRoleId-55',
+    description: 'myDescription-55'
+  }, {
+    name: 'myRole-56',
+    id: 'myRoleId-56',
+    description: 'myDescription-56'
+  }, {
+    name: 'myRole-57',
+    id: 'myRoleId-57',
+    description: 'myDescription-57'
+  }, {
+    name: 'myRole-58',
+    id: 'myRoleId-58',
+    description: 'myDescription-58'
+  }, {
+    name: 'myRole-59',
+    id: 'myRoleId-59',
+    description: 'myDescription-59'
+  }, {
+    name: 'myRole-60',
+    id: 'myRoleId-60',
+    description: 'myDescription-60'
+  }, {
+    name: 'myRole-61',
+    id: 'myRoleId-61',
+    description: 'myDescription-61'
+  }, {
+    name: 'myRole-62',
+    id: 'myRoleId-62',
+    description: 'myDescription-62'
+  }, {
+    name: 'myRole-63',
+    id: 'myRoleId-63',
+    description: 'myDescription-63'
+  }, {
+    name: 'myRole-64',
+    id: 'myRoleId-64',
+    description: 'myDescription-64'
+  }, {
+    name: 'myRole-65',
+    id: 'myRoleId-65',
+    description: 'myDescription-65'
+  }, {
+    name: 'myRole-66',
+    id: 'myRoleId-66',
+    description: 'myDescription-66'
+  }, {
+    name: 'myRole-67',
+    id: 'myRoleId-67',
+    description: 'myDescription-67'
+  }, {
+    name: 'myRole-68',
+    id: 'myRoleId-68',
+    description: 'myDescription-68'
+  }, {
+    name: 'myRole-69',
+    id: 'myRoleId-69',
+    description: 'myDescription-69'
+  }, {
+    name: 'myRole-70',
+    id: 'myRoleId-70',
+    description: 'myDescription-70'
+  }, {
+    name: 'myRole-71',
+    id: 'myRoleId-71',
+    description: 'myDescription-71'
+  }, {
+    name: 'myRole-72',
+    id: 'myRoleId-72',
+    description: 'myDescription-72'
+  }, {
+    name: 'myRole-73',
+    id: 'myRoleId-73',
+    description: 'myDescription-73'
+  }, {
+    name: 'myRole-74',
+    id: 'myRoleId-74',
+    description: 'myDescription-74'
+  }, {
+    name: 'myRole-75',
+    id: 'myRoleId-75',
+    description: 'myDescription-75'
+  }, {
+    name: 'myRole-76',
+    id: 'myRoleId-76',
+    description: 'myDescription-76'
+  }, {
+    name: 'myRole-77',
+    id: 'myRoleId-77',
+    description: 'myDescription-77'
+  }, {
+    name: 'myRole-78',
+    id: 'myRoleId-78',
+    description: 'myDescription-78'
+  }, {
+    name: 'myRole-79',
+    id: 'myRoleId-79',
+    description: 'myDescription-79'
+  }, {
+    name: 'myRole-80',
+    id: 'myRoleId-80',
+    description: 'myDescription-80'
+  } ]
+};
```

Comment:
Looks good! If you wanted to, you could replace this with a for loop to generate this at runtime, but it's up to you!